.github/workflows/ci.yml

```diff
@@ -40,11 +40,18 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Setup .NET Core SDK
-        if: runner.os != 'Windows'
+      - name: Setup .NET Core SDK v2.1.x
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 3.1.404
+          dotnet-version: 2.1.x
```

Comment:
End of support from MSFT for .NET Core 2.1 is `August 21, 2021`, less than 3 months from now. I wouldn't bother including it, personally.

Reply:
It is required because we have `netcoreapp2.1` in the test project.  We can create a new issue as a remainder to remove support for it near the end of this year.

PS:
I think it wasn't necessary to explicitly install SDK `v2.x` because SDK `v3` can target both `netcoreapp2.1` and `netcoreapp3.1`. It turns out that SDK `v5.x` cannot do that anymore.

Reply:
> I think it wasn't necessary to explicitly install SDK v2.x because SDK v3 can target both netcoreapp2.1 and netcoreapp3.1.

Then we might as well remove it? I think I had issues when running (not _building_) but I can't remember now. 

As you want, no strong feelings.

Reply:
I think we should keep testing it until it is officially supported..